vendors/marvell/boards/mw300_rd/aws_demos/application_code/main.c

```diff
@@ -171,13 +177,19 @@ int main( void )
                             tskIDLE_PRIORITY,
                             mainLOGGING_MESSAGE_QUEUE_LENGTH );
 
+    /* Initialize AWS system libraries.
+     * SYSTEM_Init() initializes mbedTLS and the
+     * random number generator and should be called
+     * before provisioning or FreeRTOS_IPInit(). */
+    SYSTEM_Init();
```

Comment:
Similar to comment above, if this has any dependency on the OS, such as using hardware for a seed, it may not work here. 

Also, in the original call site below, the return status of this function was being checked.